tensorflow/lite/kernels/activations_test.cc

```diff
@@ -243,6 +247,18 @@ class LogisticOpTest : public SingleOpTest {
   }
 };
 
+const auto kLogSoftmaxKernelMap = std::map<string, TfLiteRegistration*>({
```

Comment:
This violates trivially destructible static rules, please allocate this on the heap like the other map statics:

```
const auto kLogSoftmaxKernelMap = new std::map<string, TfLiteRegistration*>({...
```

Reply:
Oh I was wondering why there was a `new` and thought it was a memory-leak mistake, I didn't know this part of the [guideline](https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables). I also fixed https://github.com/tensorflow/tensorflow/pull/48506 where I did the same.